css/theme-topic.css

```diff
@@ -127,32 +139,32 @@
 }
 
 @media screen and (min-width: 991px) {
-.contributors li {
+.experimental .contributors li {
     display: inline-block;
     float: left;
     margin-right: .7em;
 }
 }
 
 @media screen and (min-width: 991px) {
-.contributors li:first-child:before {
+.experimental .contributors li:first-child:before {
     content: none;
 }
-.contributors li:before {
+.experimental .contributors li:before {
     content: "\2022";
     margin-right: .7em;
 }
 }
 
 @media screen and (max-width: 1200px) {
-  .contributors ul {
+ .experimental .contributors ul {
     margin-inline-start: -30px !important;
   }
 
 }
 
 @media screen and (max-width: 1200px) {
-  .contributors ul {
+ .experimental .contributors ul {
     margin-inline-start: -30px !important;
```

Comment:
Can we avoid the use of `!important` if the class name is this specific?

Reply:
Gone